src/service/pokemonService.js

```diff
@@ -0,0 +1,83 @@
+import { getPokemonById, getTypeById } from "../api/pokemon"
+
+
+// example result of calling getPokemonById(1):
+// { id: 1, name: 'Bulbasaur', photoUrl: 'someurl', types: [ { id: 3, name: 'grass' }, { id: 6, name: 'poison' } ] }
+// example result of calling getTypeById(3):
+// { id: 3, name: 'grass' }
+
+
+// example of getNextQuestion('guessName')
+// { 
+//     question: { pokemonName: 'Bulbasaur', pokemonUrl: 'someUrl', }, 
+//     answers: [ 'answer1', 'answer2', 'answer3', 'answer4', 'answer5-forTypes', 'answer6-forTypes', ], 
+//     correctAnswer: 'Bulbasaur' 
+// }
+
+class Question {
```

Comment:
I think it should be called "PokemonService" instead (to follow convention: the most important thing in fale should be named the same as file)

Reply:
or even better - change both names to "QuestionService", but also update task in Trello :)

Reply:
File and class name was changed to QuestionService. Trello task was updated.